src/core_plugins/metric_vis/public/metric_vis_params.html

```diff
@@ -195,7 +248,12 @@
 
         <div class="kuiSideBarFormRow">
           <label class="kuiSideBarFormRow__label" for="metricFontSize">
-            Font Size (<span ng-bind="editorState.params.metric.style.fontSize"></span>pt)
+            <span
+              i18n-id="metricVis.public.metricVisParams.style.fontSizeLabel"
+              i18n-default-message="Font Size"
+            >
+            </span>
+             (<span ng-bind="editorState.params.metric.style.fontSize"></span>pt)
```

Comment:
Here I didn't translate pt because it`s a unit for measurement.
![image](https://user-images.githubusercontent.com/11524637/45554841-f5bc2a00-b83f-11e8-9800-8a94cf4083c5.png)

If there are any objections, let's discuss.